pkg/services/sqlstore/temp_user.go

```diff
@@ -132,3 +133,15 @@ func GetTempUserByCode(query *models.GetTempUserByCodeQuery) error {
 	query.Result = &tempUser
 	return err
 }
+
+func ExpireOldUserInvites(cmd *models.ExpireTempUsersCommand) error {
```

Comment:
Do these get cleaned up (Deleted from db at some point?)

Reply:
Conversation was had outside of GH but following up here for completeness:

Currently in its current state with this PR, the rows will only ever be marked as expired and never deleted. The original solution a few commits back was explicitly deleting the rows, but that was since changed to be more consistent with the existing pattern based on feedback. I'm happy with either solution, but had opted for purging initially because when the temp_user goes to status `expired`, it's essentially a dead row by that point and can't/shouldn't be revived.

It would be possible to include also deleting as part of this PR - maybe the job will first mark rows as expired, and then secondly delete expired rows that haven't been updated in 1 week (or whatever)

Reply:
We can potentially add delete separately. One idea later could also be to show expired invites for tracking purpose. Would also allow to "re-send" the invite. But I'm not sure it makes sense.

Reply:
Sounds good to me. Let's revisit